### ISSUE TITLE
ofxBeacon Android

### ISSUE THREAD

**Issue by neuston** [minusplusminus/ofxBeacon#1]
Do you think ofxBeacon can be used in Android?
If not, which part of code I should change to achieve this? 
Thank you very much for any advise.

**Comment by minusplusminus** [comment]
Nope.  You have got to use JNI to bridge Java to c++. So you take an Java library like this: https://github.com/AltBeacon/android-beacon-library and then link it using this: https://developer.vuforia.com/library/articles/Solution/How-To-Communicate-Between-Java-and-C-using-the-JNI

**Comment by neuston** [comment]
Thank you. I will have a try. :)
